CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 ## Improvements
 
 - Improve complex type handling and its representation in the generated OData clients.
+- Improve collection type handling and its representation in the generated OData clients.
```

Comment:
Please correct me. I thought we agreed to not touch the change log, if the changes were about the v4 only.

Reply:
To be honest I don't know - I remember that for bigger features we always had notes in the changelog (also for the experimental stuff) but I also remember us talking about this. @FrankEssenberger what's your opinion?

Reply:
@FrankEssenberger what do you think?